flixel/FlxObject.hx

```diff
@@ -35,47 +36,47 @@ class FlxObject extends FlxBasic
 	/**
 	 * Generic value for "left". Used by `facing`, `allowCollisions`, and `touching`.
```

Comment:
These should probably have a note along the lines of this in their doc comments:

> Note: This exists for backwards compatibility, prefer using `FlxDirections.LEFT` directly.

A proper deprecation warning would probably be too annoying at this point, since these are probably used fairly commonly.